src/main/java/org/gridsuite/study/server/NetworkModificationTreeService.java

```diff
@@ -281,8 +282,8 @@ public void cloneStudyTree(AbstractNode nodeToDuplicate, UUID nodeParentId, Stud
 
             if (sourceNode instanceof NetworkModificationNode) {
                 NetworkModificationNode model = (NetworkModificationNode) sourceNode;
-                UUID modificationGroupToDuplicateId = model.getNetworkModification();
-                model.setNetworkModification(modificationGroupToDuplicateId != null ? newModificationGroupId : null);
+                UUID modificationGroupToDuplicateId = model.getNetworkModificationId();
+                model.setNetworkModificationId(modificationGroupToDuplicateId != null ? newModificationGroupId : null);
```

Comment:
What is the case where `modificationGroupToDuplicateId` is null ? is it possible to have a node without `NetworkModificationUuid` at any moment ?

Reply:
Removed